python/aida_interchange/aida_rdf_ontologies.py

```diff
@@ -5,9 +5,8 @@
 # TODO: temporarily extend these to include all ColdStart entity types - #2
 # TODO: long-term, we need to make the program ontology configurable
 
-"""
-Seedling Ontology. Currently Hardcoded
-"""
+
+# Seedling Ontology. Currently Hardcoded
```

Comment:
This comment does not show up when generating the documentation. Was that intended? If you want to it show up append `:` after the hash.

Reply:
Yes - intentional.  I removed the comment that was getting generated in the incorrect location and made the decision that these comments didn't need to be in the python api docs.  It's not really part of the API.